Gemfile

```diff
@@ -3,7 +3,7 @@ source 'https://rubygems.org'
 gem 'rails', '4.1.10'
 
 # Choose your weapon
-gem 'sqlite3'
+gem 'sqlite3', group: [:development, :test]
```

Comment:
I would at least also add the groups to the other two lines (for pg and mysql2).